libpod/image/image.go

```diff
@@ -274,22 +274,23 @@ func (i *Image) getLocalImage() (*storage.Image, error) {
 	if err == nil {
 		return repoImage, nil
 	}
+
 	return nil, errors.Wrapf(err, imageError)
 }
 
 // hasRegistry returns a bool/err response if the image has a registry in its
 // name
-func (i *Image) hasRegistry() (bool, error) {
-	imgRef, err := reference.Parse(i.InputName)
-	if err != nil {
-		return false, err
-	}
-	registry := reference.Domain(imgRef.(reference.Named))
-	if registry != "" {
-		return true, nil
-	}
-	return false, nil
-}
+//func (i *Image) hasRegistry() (bool, error) {
```

Comment:
Don't comment out, just remove.  Git will keep track of old code.

Reply:
oversight on my part ...